packages/flutter/test/rendering/mouse_tracker_test.dart

```diff
@@ -72,63 +72,81 @@ void main() {
 
   final Matrix4 translate10by20 = Matrix4.translationValues(10, 20, 0);
 
-  test('should detect enter, hover, and exit from Added, Hover, and Removed events', () {
```

Comment:
Hard to read the diff here, but all I did was wrap this in a `for` so that it's run once for mouse gestures and once for stylus.